stores.html

```diff
@@ -5,8 +5,164 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="./css/stores.css" />
```

Comment:
* You can improve CSS code reuse and manageability by extracting commonly used CSS code in a shared referenced file. CSS specific to `index.html` and `stores.html` are stores separately.